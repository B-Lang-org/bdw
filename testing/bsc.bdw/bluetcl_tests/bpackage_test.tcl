namespace import ::Bluetcl::*

# Loads the specified package
puts [bpackage load Test]
puts ""

# Loads the specified package
puts [bpackage load RegFile]
puts ""

# Lists currently loaded packages
puts [bpackage list]
puts ""

# Shows package dependencies
puts [bpackage depend]
puts ""

# Clears all currently loaded packages
puts [bpackage clear]
puts ""

# Lists currently loaded packages(list should be empty)
puts [bpackage list]

