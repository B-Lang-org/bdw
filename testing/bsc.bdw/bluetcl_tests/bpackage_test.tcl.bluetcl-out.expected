Test Vector Array List PreludeBSV Prelude

Test Vector RegFile Array List ConfigReg PreludeBSV Prelude

Test Vector RegFile Array List ConfigReg PreludeBSV Prelude

{Test {Prelude PreludeBSV List Array Vector}} {Vector {Prelude PreludeBSV List Array}} {RegFile {Prelude PreludeBSV ConfigReg List}} {Array {Prelude PreludeBSV List}} {List {Prelude PreludeBSV}} {ConfigReg {Prelude PreludeBSV}} {PreludeBSV Prelude} {Prelude {}}




